api/src/main/resources/workbench.yaml

```diff
@@ -141,38 +141,61 @@ paths:
         "204":
           description: Account deleted successfully.
 
-  /api/v1/register:
+  /api/v1/id-verification:
```

Comment:
nit: existing issue, but I'd generally recommend naming custom REST actions with verb names, e.g. verify-id, accept-terms-of-service

Reply:
Done.